keps/sig-instrumentation/Graduate metrics.k8s.io API to GA/kep.yaml

```diff
@@ -0,0 +1,22 @@
+id: ""
+prnumber: "5208"
+name: Graduate metrics.k8s.io API to GA
+title: Graduate metrics.k8s.io API to GA
```

Comment:
```suggestion
name: API for internal metrics integration
title: API for internal metrics integration
```

Reply:
"internal metrics" ? What do you mean ?

Reply:
We can pick a different name; the key thing is to drop "Graduate" from the name of the enhancement.

Reply:
We've called it "graduate ..." on KEPs that are laying out a new plan to stabilize an API.

E.G. 

https://github.com/kubernetes/enhancements/tree/master/keps/sig-autoscaling/2702-graduate-hpa-api-to-GA
https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/19-Graduate-CronJob-to-Stable
https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/2334-graduate-server-side-get-and-partial-objects-to-GA/README.md
https://github.com/kubernetes/enhancements/tree/master/keps/sig-api-machinery/2338-graduate-API-gzip-compression-support-to-GA
...

Reply:
On several of those, I've tried to intervene to fix what I see as problematic KEP naming.

https://github.com/kubernetes/enhancements/pull/5208#issuecomment-2757242737

Reply:
KEP-19 for example is now titled **CronJobs (previously ScheduledJobs)**